pytest.ini.example

```diff
@@ -23,3 +23,4 @@ markers =
     rpsaas: mark tests that are purely cloud-side
     e2e: mark tests that run in e2e pipeline
     upgrade_scenario_test: mark tests that will run az iot ops upgrade
+    secretsync_test: mark tests that will run az iot ops secretsync
```

Comment:
FWIW all of these are 'test' marks so i feel like `_test` is a bit redundant

Are there other tests that will eventually run as part of this "scenario"? Wondering if there's a broader term we should use for this mark that signifies it needs WLIF etc rather than just secretsync - but it's probably fine for now.

Reply:
I can change it to wlif - its prob a better marker